app/models/city.rb

```diff
@@ -1,4 +1,9 @@
 class City < ActiveRecord::Base
+  CITIES = [
+    DENVER = 'Denver',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.